src/main/java/roomescape/controller/AdminPageController.java

```diff
@@ -0,0 +1,30 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("admin")
```

Comment:
`/`가 빠져 있네요!
없어도 잘 동작은 하지만, 일반적으로 붙이는 것이 관행입니다.
해당 내용 전반적으로 체크해주세요~

```suggestion
@RequestMapping("/admin")
```

Reply:
오 그렇군요..!!
붙여서 수정하였습니다 👍